src/app/views/query-runner/request/Request.tsx

```diff
@@ -19,13 +21,51 @@ import { translateMessage } from '../../../utils/translate-messages';
 import { convertVhToPx } from '../../common/dimensions-adjustment';
 import { Auth } from './auth';
 import { RequestBody } from './body';
+import { loadAndInitialize } from './feedback/feedbackWrapper';
 import { RequestHeaders } from './headers';
 import { Permission } from './permissions';
 import './request.scss';
 
 export class Request extends Component<IRequestComponent, any> {
   constructor(props: IRequestComponent) {
-    super(props);
+    super(props)
+    this.state = {
+      officeBrowserFeedback: undefined,
+      enableShowSurvey: false,
+    }
+    // this.setOfficeBrowserFeedbackUtility = this.setOfficeBrowserFeedbackUtility.bind(this);
+    this.showCustomSurvey = this.showCustomSurvey.bind(this);
+    this.initializeFeedback();
+  }
+  initializeFeedback() {
+    const floodgateObject = makeFloodgate();
+    loadAndInitialize(floodgateObject).then(() => {
+      this.setState({
+        officeBrowserFeedback: floodgateObject,
+        enableShowSurvey: true,
+      })
+    });
+  }
+
+  private showCustomSurvey() {
+    const customSurvey: OfficeBrowserFeedback.ICustomSurvey = {
+      campaignId: "10000000-0000-0000-0000-000000000000",
```

Comment:
Can we have this in an env file?

Reply:
Should also be added to CI/CD env variables otherwise the built files won't have this Id. Is that the right Id? It looks like a dummy value.

Reply:
@jobala it was a dummy value...
But once I get the correct value I will add them